contracts/CourtRoom.sol

```diff
@@ -37,7 +45,6 @@ contract CaseContract is Owned {
 		return clm.claimId;
 	}
```

Comment:
Unless the same address (_plaintiff) submitted the same evidence multiple times on the same block, claimId would return a different value each time, meaning that the require would always return true, no?